doc/man1/openssl-speed.pod.in

```diff
@@ -112,9 +111,18 @@ pre-compiled grand selection is tested.
 
 =back
 
+=head1 BUGS
+
+The I<algorithm> can be selected only from a pre-compiled subset of things
+that the C<openssl speed> command knows about. To test any additional digest
+or cipher algorithm supported by OpenSSL use the C<-evp> option.
+
+There is no way to test speed of additional public key algorithms supported
```

Comment:
_... to test **the** speed of **any** additional ..._

Reply:
Fixed.